<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C++ RPG Game</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="Main">
       <widget class="QPushButton" name="startGameBtn">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>270</y>
          <width>461</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Game</string>
        </property>
       </widget>
       <widget class="QPushButton" name="exitGameBtn">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>300</y>
          <width>461</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Exit Game</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>330</y>
          <width>161</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>created by @norcholly</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="InputName">
       <widget class="QLineEdit" name="nameInput">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>280</y>
          <width>321</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="yourWarriorsName">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>280</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Your Warrior's Name:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="startWithNameBtn">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>280</y>
          <width>88</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Battle">
       <widget class="QTextBrowser" name="consoleArea">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>782</width>
          <height>440</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="retreatBtn">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>580</y>
          <width>782</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Retreat</string>
        </property>
       </widget>
       <widget class="QPushButton" name="attackBtn">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>516</y>
          <width>782</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Attack</string>
        </property>
       </widget>
       <widget class="QPushButton" name="healBtn">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>548</y>
          <width>782</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Drink Heal Potion</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="hpShowOnBattle">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>385</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="goldShowOnBattle">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>20</y>
          <width>385</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Tutorial">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextBrowser" name="tutorialArea"/>
        </item>
        <item>
         <widget class="QPushButton" name="continueBtn">
          <property name="text">
           <string>Continue</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Game">
       <widget class="QTextBrowser" name="levelShow">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>90</y>
          <width>785</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="goToBattleBtn">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>424</y>
          <width>785</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Go to Battle</string>
        </property>
       </widget>
       <widget class="QPushButton" name="inventoryBtn">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>488</y>
          <width>785</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Inventory</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="goldShow">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>240</y>
          <width>785</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="dpShow">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>190</y>
          <width>785</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="suicideBtn">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>552</y>
          <width>785</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Suicide</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="nameShow">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>40</y>
          <width>785</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="hpShow">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>140</y>
          <width>785</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="sleepBtn">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>520</y>
          <width>785</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Sleep</string>
        </property>
       </widget>
       <widget class="QPushButton" name="shopBtn">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>456</y>
          <width>785</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Shop</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="scoreShow">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>290</y>
          <width>785</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="dayShow">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>340</y>
          <width>785</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Inventory">
       <widget class="QTextBrowser" name="invSlot_3">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>100</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="invSlot_2">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>100</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="invSlot_1">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>100</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="invSlot_0">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="invSlot_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="invSlot_5">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>150</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="invSlot_6">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>150</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="invSlot_7">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>150</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="invSlot_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="invSlot_9">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>200</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="invSlot_10">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>200</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="invSlot_11">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>200</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="invSlot_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>250</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="invSlot_13">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>250</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="invSlot_14">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>250</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="invSlot_15">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>250</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="turnBackBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>1031</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Turn to Back</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Shop">
       <widget class="QPushButton" name="buyDpBtn">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>350</y>
          <width>451</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Enhance your Sword</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buyPotionBtn">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>270</y>
          <width>451</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Health Potion</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="dpArea">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>350</y>
          <width>291</width>
          <height>31</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu Sans'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;50G (Permanent +20DP)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="backBtn">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>230</y>
          <width>451</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Turn to Back</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="yourGoldArea">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>230</y>
          <width>291</width>
          <height>31</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu Sans'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buyHpBtn">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>310</y>
          <width>451</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Enhance your Armor</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="hpArea">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>310</y>
          <width>291</width>
          <height>31</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu Sans'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;70G (Permanent +20HP)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="ptArea">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>270</y>
          <width>291</width>
          <height>31</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu Sans'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;20G (Single Use +60HP)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Suicide">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextBrowser" name="scoreArea"/>
        </item>
        <item>
         <widget class="QPushButton" name="mainMenuBtn">
          <property name="text">
           <string>Main Menu</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
